creator-node/src/utils.js

```diff
@@ -71,6 +71,36 @@ async function getIPFSPeerId (ipfs, config) {
   return ipfsIDObj
 }
 
+async function ensureMultihashPresent (ipfs, multihash, storagePath) {
+  // Confirm availability by catting single byte locally
+  let timeoutPromise = new Promise((resolve, reject) => {
+    let wait = setTimeout(() => {
```

Comment:
@hareeshnagaraj you could re-write this as like 
```const wait = (ms) => new Promise((resolve, reject) => setTimeout(() => reject(new Error('Timeout')), ms))```

Reply:
👍